dashboard/test/factories.rb

```diff
@@ -1,4 +1,10 @@
-FactoryGirl.define do
+FactoryGirl.define do  factory :school_district do
+    name "MyString"
+city "MyString"
+state "MyString"
+zip "MyString"
```

Comment:
Please make these strings distinct to detect field mixups and make debugging easier.